frontend/pages/hosts/ManageHostsPage/ManageHostsPage.jsx

```diff
@@ -635,10 +642,9 @@ export class ManageHostsPage extends PureComponent {
       currentUser,
       selectedFilter,
       selectedLabel,
-      hosts,
       loadingHosts,
```

Comment:
Does this need to be replaced with something in `state` to handle the loading state?

Reply:
Working on it. I didn't notice it before.